site/assets/scss/_variables.scss

```diff
@@ -1,5 +1,5 @@
 // Local docs variables
-$bd-primary:       $primary; // Boosted mod
+$bd-primary:       $brand-orange; // Boosted mod
```

Comment:
I'm not sure about this change. It's used for instance in toc border left color that should be darker orange in light mode. Have you seen other use cases needing this change?

Reply:
It can't work right now because of `lighten`, etc. so I've put back the original value which is `$primary`.